R/key-helpers.R

```diff
@@ -62,6 +93,15 @@ is_unique_key <- nse_function(c(.data, column), ~ {
   duplicate_rows
 })
 
+is_key <- nse_function(c(.data, column), ~ {
+  col_expr <- ensym(column)
+  col_name <- as_name(col_expr)
+
+  duplicate_rows <- .data %>%
+    count(value = !!col_expr) %>%
+    filter(n != 1)
```

Comment:
Does this work for remote data sources?

Reply:
Actually, this function isn't used anymore in this version, so I deleted it. Thanks for drawing attention to it.

Regarding your question: the function `is_unique_key()` uses the same logic (it is used for `enum_pk_candidates()`). I just verified that it does work on Postgres.